### PR TITLE
Update HA documentation page (from README.md source)

### PR DESCRIPTION
**Pull request by tmack8001**
Didn't initially realize the info.md and README.md were different

### CODE REVIEW

**Comment by jrgutier** [comment]
Corrected grammar also. Please reread, just in case I messed something up.

**Review by jrgutier — APPROVED** [review]

**Comment by tmack8001** [comment]
I synced the changes you made over into the README.md as well since the data contained in these two files is essentially the same/similar.

File: info.md [hunk 1]
```diff
@@ -36,6 +43,24 @@ Your vehicle must be in delivered status and in your posession for this integrat
 
 {% endif %}
 
+## Configuration
+
+To connect your newly installed integration to pull available sensor data (see the below example) for your Rivian vehicle, you should follow the following configuration recommendations. Not doing so could put your main Rivian account at risk of being locked and available.
+
+* One Time Password (**recommended**) - It is best security practice always to enable One Time Password / Multi-Factor Authentication. This way, your credentials (username + password) can not be used without your explicit consent through the personal device configured on your Rivian account.
```
Comment (bretterer) [inline info.md:50]: Until we have this working, we should leave this recommended step out

Comment (tmack8001) [reply info.md:50]: It works now and closed the issue tracking it as well 😜

**Pull request merged**